strawberry/utils/typing.py

```diff
@@ -103,7 +103,7 @@ def is_type_var(annotation: Type) -> bool:
     return isinstance(annotation, TypeVar)
 
 
-def get_parameters(annotation: Type):
+def get_parameters(annotation: Type) -> Union[Any, Tuple[()]]:
```

Comment:
I think this might be tuple of objects

Reply:
It should be `Union[Tuple[object], Tuple[()]]` then, otherwise mypy will break because of the `return ()` in this function